### PR TITLE
Easier local setup of mrfgen tools

### PR DESCRIPTION
**Pull request by zmoon**
The main change is the addition of a Conda env spec and some edits to the mrfgen readme.

I also wrote a small install script and made some minor changes in order for `--help` to work without error for all of the tools. `src/mrfgen/RgbPngToPalPng.py` had some `\r`s in it, I ran `dos2unix` on it to get it to work as an executable. I also linked `oe_utils.py` since it's needed.

I wasn't able to get all the same versions from conda-forge as pinned elsewhere, so open to thoughts about that.

### CODE REVIEW

File: src/mrfgen/RgbPngToPalPng.py [hunk 1]
```diff
@@ -1,114 +1,114 @@
...
+        # Check it is .png file and it exists.
+        fileExist = os.path.exists(rgbaPng)
+        if '.png' in rgbaPng.lower() and fileExist:
+            print('Input RGB PNG image: ' + rgbaPng)
+        else:
+            print('Input image is not PNG file or does not exist.')
+            sys.exit(-1)
+    elif opt == '-o':
+        palPng = arg
+        print('Output pelette PNG image: ' + palPng)
```
Comment (lucianpls) [inline src/mrfgen/RgbPngToPalPng.py:86]: spelling, "palette"

Comment (zmoon) [reply src/mrfgen/RgbPngToPalPng.py]: 👍

**Comment by jtroberts** [comment]
Thanks for the contribution! We'll run some test with GIBS to make sure our builds are happy.

We've had some prior discussions about making mrfgen its own standalone repository. I'd be interested to hear if that would be a worthwhile pursuit.

**Comment by zmoon** [comment]
> making mrfgen its own standalone repository

Yes I did also think about this. It would be convenient if it had a conda-forge recipe, say. And that would be much easier to set up if mrfgen had its own repo.

I was also curious about the status of https://github.com/nasa-gibs/gpu-data-to-image

**Comment by jtroberts** [comment]
> I was also curious about the status of https://github.com/nasa-gibs/gpu-data-to-image

It was intended as a prototype and has been dormant for a few years. It worked great back then and could be revived. We wanted to integrate it with https://github.com/apache/sdap-nexus but never had the cycles to put things together.

**Review by grabermtw — APPROVED** [review]
Thank you!

**Pull request merged**